client/my-sites/upgrades/components/form/hidden-input.jsx

```diff
@@ -14,14 +14,17 @@ module.exports = React.createClass( {
 
 	componentWillReceiveProps: function( nextProps ) {
 		if ( ! this.state.toggled && ! isEmpty( nextProps.value ) ) {
-			this.setState( { toggled: true } );
+			this.setState( {
+				toggled: true,
+				autofocus: false
```

Comment:
Why do we have an `autofocus` in state? Can we do it with adding a `ref` to `<Input />` and call `this.refs.focus()`?